packages/amplify-graphql-model-transformer/rds-lambda/tsconfig.json

```diff
@@ -7,7 +7,8 @@
     "target": "es6",
     "outDir": "lib",
     "moduleResolution": "node",
-    "lib": ["es2015"],
+    "module": "CommonJS",
```

Comment:
Can we use `module`?

Reply:
because aws lambda has some limitations these were the only configs I was able to use to get it to compile and run correctly, it is of type module in the package.json